docs/backup-restore.md

```diff
@@ -90,7 +90,8 @@ Specifically, please follow the steps below.
    * Remove table A
    * Create a table named A with backup B
 
-Note that tables can only be restored one by one so you need to do the above steps for each table.
+Note:
 
-Note also that some of the configurations such as PITR and auto scaling policies are reset to the default values for restored tables, so please manually configure required settings. See [the doc](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CreateBackup.html#CreateBackup_HowItWorks-restore) for more detail.
-If you used the schema loader and didn't change any other configurations, executing the schema loader with the same schema file is one of the ways to restore the original configuration. It only sets missing configurations if a specified table exists.
+* You can create a backup for the `scalardb.metadata` table directly, which does not need to be restored with PITR.
```

Comment:
What `directly` means here? Are we discussing a direct way and an indirect way?
Also, the reason for not needing to be restored with PITR should be mentioned, otherwise, readers (including me) don't really get what it means.

Reply:
> What directly means here?

Sorry for the inconvenience.
Directly means that we can create a backup of scalardb.metadata without restoring the PITR.

> Are we discussing a direct way and an indirect way?

No, we need to follow the steps of `Create a backup` if we want to restore from PITR.
But in the case of the `scalardb.metadata` table, we can restore it without the PITR because there is no change after it is created.

I will update the PR.

Reply:
Document updated with the following sentence
```
You can create a backup for the `scalardb.metadata` table without restoring the PITR because it contains metadata of the schemas (It will not change after the schema creation).
```